tools/licenses/README.md

```diff
@@ -20,9 +29,26 @@ rm -rf ../../../out/licenses
 dart lib/main.dart --src ../../.. --out ../../../out/licenses --golden ../../ci/licenses_golden
 ```
 
-Then copy any affected files from `../../../out/licenses` to `../../ci/licenses_golden` and add them to your change.
+Then, copy any affected files from `../../../out/licenses` to
+`../../ci/licenses_golden` and add them to your change, and examine
+the diffs to see what changed.
+
+```
+cp ../../../out/licenses/* ../../ci/licenses_golden
+git diff
+```
+
+If the only changes are to what files are included, then you're good
+to go. However, if any of the _licenses_ changed, whether new licenses
+are added, old ones removed, or any have their content changed in any
```

Comment:
I would emphasize the "in any way" part.  E.g. people might assume whitespace changes don't constitute a "change".

Reply:
updated README for this and other general improvements